packages/augur-artifacts/src/configuration.ts

```diff
@@ -205,7 +205,8 @@ export const DEFAULT_SDK_CONFIGURATION: SDKConfiguration = {
     autoReport: false,
   },
   uniswap: {
-    exchangeRateBufferMultiplier: 1.05,
+    // mainnet will be <= 1.05 but for dev we can get away with a wide spread
+    exchangeRateBufferMultiplier: 1.75,
```

Comment:
can the value be moved to network config files since they will be different per network?

Reply:
It is there. This is just the default. The mainnet.json file uses 1.05

Reply:
It seems like the default should be mainnet numbers, and the config should override that.  That way if config fails to be applied for some reason, you don't fail over to "users get robbed blind".